apps/namadillo/src/utils/index.ts

```diff
@@ -33,11 +34,16 @@ export const epochToString = (epoch: bigint): string =>
 export const proposalIdToString = (proposalId: bigint): string =>
   `#${proposalId.toString()}`;
 
-export const addTransactionEvent = <T>(
+export const useTransactionEventListener = <T>(
```

Comment:
There is also `useEventListener` in `@namada/hooks` which does almost the same as this function, but I think either way is fine.

Reply:
Thanks @emccorson ! I didn't know it, good to know!

I tried to use the hook, but the typescript change become a big lift. So I'll merge this PR as it's and prepare it in another PR, so we have a better visibility over the changes and the refactor proposal